src/redux/usersPositions.ts

```diff
@@ -58,14 +63,29 @@ interface Position {
   token1PriceUSD: number;
 }
 
-export type StoredPositions = Position &
+export type StoredPosition = Position &
```

Comment:
Is it worth renaming these types to `*UniswapPosition*` assuming we will have other sources in the future?

Reply:
good idea!